float32[] targetjoint
---
sensor_msgs/CompressedImage result